### DISCUSSION TITLE
Github Pages content doesn’t update

### DISCUSSION THREAD

**Discussion by MTiryaki** [community/community#21699] [Other Feature Feedback, Questions, & Ideas]
<p>My Github Pages page’s content fails to update, even though the repo shows the new content in the repo.</p>

**Comment by foopod** [comment, marked as answer]
<p>Seems to be up to date now. Your latest visible <a href="https://github.com/MTiryaki/MTiryaki.github.io/commit/5c582b883fe6bea85edb4b707f401512c083d650" rel="nofollow noopener">change</a>appears is live on your site.</p>
